en/developer_guide/block_parameter.rst

```diff
@@ -0,0 +1,126 @@
+Block parameter
+===============
+
+Context
+-------
+
+To be properly displayed on the website, some blocks will require environment parameters.
+Those parameters could be of multiple types, for instance :
+
+ - url parameters
+ - server parameters
+ - header parameters
+ - ...
+
+An easy way to give those parameters to the block would have been to give all the request
+parameters when the block is called.
+
+This easy solution is not optimal for a main reason: when loading blocks via `ESI`_, each
+block will be cached for each url variant.
+
+If the block 1 does not require any parameters, in the following exemple :
+
+ * ``/block/1`` will be in one cache entry.
+ * ``/block/1?foo=bar`` will be in another cache entry.
+
+To improve your cache hit ratio, it will be likely to reduce the number of cache
+entry for the same page.
```

Comment:
"it will be likely..." => "you should have the least entries for a given page" or something like this."it will be likely" does not fit here.